editor/editor_spin_slider.h

```diff
@@ -85,6 +85,7 @@ class EditorSpinSlider : public Range {
 	void _grabber_mouse_entered();
 	void _grabber_mouse_exited();
 	void _focus_entered();
+	void _value_input_gui_input(const Ref<InputEvent> &p_event);
```

Comment:
Is there a reason this is `protected` and not `private` like the other `_value_input` methods?

Reply:
I think I lumped it with the spin slider's `_gui_input`, but yeah, would make more sense with the other signals. I'll move it.